rootfs-builder/rootfs.sh

```diff
@@ -16,6 +16,9 @@ AGENT_BIN=${AGENT_BIN:-kata-agent}
 AGENT_INIT=${AGENT_INIT:-no}
 KERNEL_MODULES_DIR=${KERNEL_MODULES_DIR:-""}
 
+lib_file="${script_dir}/../scripts/lib.sh"
+[ -e "$lib_file" ] && source "$lib_file" || true
```

Comment:
Now we depend on this file I wonder if would be better not add '|| true'

Reply:
Good catch - fixed!